app/components/candidate_interface/other_qualifications_review_component.html.erb

```diff
@@ -16,5 +16,14 @@
 <% end %>
 
 <% if @qualifications.empty? %>
-  <p class="govuk-body">No other qualifications entered.</p>
+  <% if @submitting_application && !application_form.other_qualifications_completed %>
+    <%= render(CandidateInterface::IncompleteSectionComponent.new(section: application_form.international? ? :other_qualifications_international : :other_qualifications, section_path: candidate_interface_other_qualification_type_path, error: @missing_error)) %>
+  <% elsif @submitting_application && application_form.other_qualifications_completed %>
+    <%= render(SummaryCardComponent.new(rows: no_qualification_row, editable: @editable)) %>
+  <% else %>
+    <div class="govuk-inset-text govuk-!-width-two-thirds govuk-!-margin-top-0">
+      <p>Adding A levels and other qualifications makes your application stronger. They demonstrate subject knowledge not covered in your degree or work experience.</p><p class="govuk-hint">Training providers usually ask you for them later in the process.</p>
```

Comment:
```
<p class="govuk-body">Adding A levels and other qualifications makes your application stronger. They demonstrate subject knowledge not covered in your degree or work experience.</p>

<p class="govuk-hint">Training providers usually ask you for them later in the process.</p>
```

Reply:
Actually, please use `govuk-body` for both paragraphs. I think this is a bug in the prototype.